.github/workflows/dagster.yml

```
@@ -58,6 +58,11 @@ on:
         default: "1.5.12"
         required: false
         # Chart values https://github.com/dagster-io/dagster/blob/1.5.12/helm/dagster/values.yaml
+      enable_sentry_release_creation:
```

Keep naming consistent?
```suggestion
      create_sentry_release:
```